kolibri/core/auth/test/test_bulk_import.py

```diff
@@ -15,8 +15,12 @@
 from kolibri.core.auth.constants import role_kinds
 from kolibri.core.auth.models import Classroom
 from kolibri.core.auth.models import FacilityUser
-from kolibri.core.utils.csv import open_csv_for_reading
-from kolibri.core.utils.csv import open_csv_for_writing
+
+file_storage = DefaultStorage()
+
+
+def random_filename():
```

Comment:
In hindsight, I think that maybe just cleaning up the files after each test could have avoided needing this.